pkgs/by-name/em/emacspeak/package.nix

```diff
@@ -1,34 +1,37 @@
 { lib
-, stdenv
+, emacs
```

Comment:
Directly pulling in `emacs` has some issues:
- the unneeded `emacs` is in the closure of `emacs-nox.pkgs.emacspeak`
- native compilation results of `emacs-nox.pkgs.emacspeak`, if generated, will not be used by `emacs-nox`

---

BTW, the directory structure of `$out` is probably wrong because it is quite different from other packages in `emacsPackages`.